agent/initialization/start.go

```diff
@@ -0,0 +1,12 @@
+package initialization
+
+import (
+	"fmt"
+
+	"github.com/kubeshop/tracetest/agent/config"
+)
+
+// Start the agent with given configuration
+func Start(config config.Config) {
+	fmt.Println("Starting agent")
+}
```

Comment:
In the server, we would call this function to start the agent along with the server

Reply:
It will be. When I say `start the agent` is literally starting the process. As we have to have the agent running along with the tracetest server, we have to include everything in the same process as you cannot run two different things in a docker container

Reply:
Oh makes sense!